run/connect_cloud_sql/main.tf

```diff
@@ -141,61 +141,53 @@ resource "google_secret_manager_secret_iam_member" "secretaccess_compute_dbname"
 
 
 # [START cloudrun_service_cloudsql_default_service]
-resource "google_cloud_run_service" "default" {
```

Comment:
Does this need to specify a cloud sql instance like using `--add-cloudsql-instances`? Do we need the sql volume https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_service#cloud_sql_instance?

Reply:
The current sample https://cloud.google.com/sql/docs/mysql/connect-run#terraform opts to add the the config as a separate step. This sample needs more work to adapt to this, or update docs to all-in-one the config

Reply:
ref https://github.com/terraform-google-modules/terraform-docs-samples/pull/168#issuecomment-1264084204, pending internal discussion

Reply:
Will address in followup PR